requirements.txt

```diff
@@ -1,6 +1,5 @@
 cmake-setuptools
```

Comment:
we no longer rely on `cmake-setuptools`
could also include tifffile here, although it's installed with other dependencies

Reply:
I think it woudl be better to make it explicit. Will have a go at removing cmake-setup tools and adding tifffile.